### ISSUE TITLE
megalencephaly of the temporal lobes

### ISSUE THREAD

**Issue by azankl** [obophenotype/human-phenotype-ontology#7384]
**Preferred term label:**
megalencephaly of the temporal lobes

**Synonyms**
temporal lobe megalencephaly
enlarged temporal lobes

**Definition (free text, please give PubMed ID)**
Expansion of the temporal lobes along all axes, resulting in a globular appearance of the brain
[PMID: 19098178](https://www.ncbi.nlm.nih.gov/pubmed/19098178) (Fig. 2)
[PMID:24585534](https://www.ncbi.nlm.nih.gov/pubmed/24585534)

**Parent term (use hpo.jax.org/app)**
temporal lobe dysplasia (new term), under Abnormal cerebral morphology HP:0002060
Megalencephaly HP:0001355

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Thanatophoric dysplasia type I (https://www.omim.org/entry/187600)
Thanatophoric dysplasia type II (https://www.omim.org/entry/187601)

**Your nano-attribution (ORCID)**
Andreas Zankl 0000-0001-8612-1062

**Comment by pnrobinson** [comment]
added both terms, thanks!
